### ISSUE TITLE
Document the fact that on AWS functions are created with a version

### ISSUE THREAD

**Issue by eladb** [winglang/wing#1405]
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### Feature Spec

document this https://github.com/winglang/wing/pull/1373

We currently don’t have documentation for resources, so there’s no place to write this.

### Use Cases

Make sure people understand the implications 

### Component

SDK, Documentation

**Comment by staycoolcall911** [comment]
@eladb - is documenting it within the [Wing SDK Spec](https://docs.winglang.io/reference/wingsdk-spec) good enough? If not, then what kind of documentation did you have in mind?

**Comment by eladb** [comment]
No, I think we need a dedicated "readme" for each resource. Opened this issue to discuss and track: https://github.com/winglang/wing/issues/1438
